reporter/src/main/kotlin/reporters/evaluatedmodel/EvaluatedModelMapper.kt

```diff
@@ -306,6 +307,11 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
             detectedLicenses = detectedLicenses,
             detectedExcludedLicenses = detectedExcludedLicenses,
             concludedLicense = pkg.concludedLicense,
+            effectiveLicense = input.licenseInfoResolver.resolveLicenseInfo(pkg.id).filterExcluded().effectiveLicense(
+                LicenseView.CONCLUDED_OR_DECLARED_AND_DETECTED,
```

Comment:
BTW, I just checked all callers of `effectiveLicense()`, and we currently sometimes use `LicenseView.CONCLUDED_OR_DECLARED_AND_DETECTED` (like here), but sometimes `LicenseView.ALL`. Would you mind double-checking whether that makes sense?

Reply:
We only use `LicenseView.ALL` in the `DownloaderCommand` which is a special case I believe, and in tests. `LicenseView.CONCLUDED_OR_DECLARED_AND_DETECTED` is what we consider the default and use in several other places, especially the static HTML reporter. We might want to consider making the default configurable later on.